docs/IntegrationTest.md

```diff
@@ -0,0 +1,175 @@
+    ```
+
+1. Make sure that `kubectl` is installed:
+    ```bash
+    kubectl version
+    ```
+
+    You can install through the [Google Cloud SDK](
+    https://kubernetes.io/docs/tasks/tools/install-kubectl/#download-as-part-of-the-google-cloud-sdk)
+    (which should also be installed on Cloud Shell):    
+    ```bash
+    gcloud components install kubectl
+    ```
+
+    If not, choose one of the other installation methods
+    [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+
+1. Create a GKE cluster similar to step 1 of [Configure GKE Cluster](Home.md#configure-gke-cluster):
+    ```bash
+    export CLUSTER=jenkins-master
+    export ZONE=us-central1-c
+    gcloud container clusters create --zone $ZONE $CLUSTER
+    ```
+
+1. Create a [domain](https://domains.google/) for hosting your jenkins instance. Export that now:
+    ```bash
+    export DOMAIN=example.com
+    ```
+
+1. Create a global [static external IP address](
+ https://cloud.google.com/compute/docs/ip-addresses/reserve-static-external-ip-address#reserve_new_static):
+    ```bash
+    export ADDRESS_NAME=jenkins-address
+    gcloud compute addresses create $ADDRESS_NAME --global
+    gcloud compute addresses describe $ADDRESS_NAME --global
+    ```
+
+    The A record for the domain you created should point to the IP address that the describe command
+    outputs.
+
+1. Create a Google-managed SSL certificate resource as described [here](
+https://cloud.google.com/load-balancing/docs/ssl-certificates#create-managed-ssl-cert-resource):
+    ```bash
+    export CERTIFICATE_NAME=jenkins-host-certificate
+    gcloud beta compute ssl-certificates create $CERTIFICATE_NAME --domains $DOMAIN
+    ```
+
+    Don't worry about any of the other steps, as the helm installation process will take care of
+    that.
+    
+1. If you haven't already, clone the google-kubernetes-engine-plugin repository on your Cloud Shell
+instance. This contains some of the resources needed.
+    ```bash
+    git clone https://github.com/jenkinsci/google-kubernetes-engine-plugin.git
+    ```
+
+    <!--TODO(stephenshank): Create publically available docker image -->
+1.  Create a Jenkins agent image, using the [Dockerfile](resources/helm/Dockerfile) from the
+documentation resources:
+    ```bash
+    cp docs/resources/helm/Dockerfile Dockerfile
+    docker build -t us.gcr.io/$PROJECT/jenkinsagent
+    docker push us.gcr.io/$PROJECT/jenkinsagent
+    ```
+
+    More detailed instructions are available
+    [here](https://cloud.google.com/container-registry/docs/pushing-and-pulling).
+
+### Helm Setup
+
+1. Choose a helm version and export it as `$HELM_VERSION`:
+    ```$bash
+    export HELM_VERSION=2.13.0
+    ```
+
+1. Run the following commands from the Cloud Shell CLI to install the
+[helm](https://helm.sh/docs/using_helm/#from-the-binary-releases) client:
+    ```bash
+    wget "https://storage.googleapis.com/kubernetes-helm/helm-v$HELM_VERSION-linux-amd64.tar.gz"
```

Comment:
I think linking to helm's installation docs is sufficient. Duplicating their installation instructions here adds to our maintenance burden.